nixos/modules/services/development/nixseparatedebuginfod.nix

```diff
@@ -0,0 +1,94 @@
+{ pkgs, lib, config, ... }:
+let
+  cfg = config.services.nixseparatedebuginfod;
+  url = "127.0.0.1:${toString cfg.port}";
+  recentNix = lib.lists.findFirst
```

Comment:
I'd highly recommend to use a proper assert and a `cfg.package` rather than a "too smart" logic that could be surprising to the user.

Reply:
done